database/migrations/2018_08_08_102805_create_post_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePostTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+          $table->increments('id');
+          $table->integer('user_id')->nullable();
+          $table->string('name')->nullable();
+          $table->text('description')->nullable();
+          $table->timestamps();
```

Comment:
If the `user_id`, `name` and `description` are all required fields via authentication or validation, then we would not make them `nullable()` as well.

Reply:
Thanks @stevethomas , I forgot to remove `nullable()` because I implemented the form validation late when I read the bonus points